tests/std/tests/GH_001596_adl_proof_algorithms/test.compile.pass.cpp

```diff
@@ -121,6 +144,48 @@ void test_algorithms() {
 
     (void) std::search(varr, varr, varr, varr);
     (void) std::search(iarr, iarr, iarr, iarr, validating_equal{});
+
+    (void) std::min(+varr, +varr);
+    (void) std::min(+iarr, +iarr, validating_less{});
+    (void) std::min({+varr, +varr});
+    (void) std::min({+iarr, +iarr}, validating_less{});
+
+    (void) std::max(+varr, +varr);
+    (void) std::max(+iarr, +iarr, validating_less{});
+    (void) std::max({+varr, +varr});
+    (void) std::max({+iarr, +iarr}, validating_less{});
+
+    (void) std::minmax(+varr, +varr);
+    (void) std::minmax(+iarr, +iarr, validating_less{});
+    (void) std::minmax({+varr, +varr});
+    (void) std::minmax({+iarr, +iarr}, validating_less{});
+
+    (void) std::minmax(+varr, +varr);
+    (void) std::minmax(+iarr, +iarr, validating_less{});

```

Comment:
These two lines are exact duplicates of lines above, but the following two lines aren't. What was intended here?

Reply:
The intent is explicitly using `+` to produce the result of decay. I must have mistakenly written these lines again.